pages/dkp/konvoy/2.1/choose-infrastructure/aks/aks-advanced/aa-new/index.md

```diff
@@ -57,6 +57,8 @@ Before you start, make sure you have completed the steps in [Bootstrap][bootstra
     azuremanagedmachinepool.infrastructure.cluster.x-k8s.io/mpd56zd created
     ```
 
+   <p class="message--important"><strong>IMPORTANT: </strong>Not all released versions of Kubernetes are available in Azure when creating a Managed Clusters (AKS). Use the Azure portal or CLI to <a href="https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#azure-portal-and-cli-versions">find the versions supported in your cluster's location</a>.</p>
```

Comment:
```suggestion
   <p class="message--important"><strong>IMPORTANT: </strong>Not all released versions of Kubernetes are available in Azure when creating a Managed Cluster (AKS). Use the Azure portal or CLI to <a href="https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#azure-portal-and-cli-versions">find the versions supported in your cluster's location</a>.</p>
```

Reply:
Why can't we just tell them the versions?

Reply:
The versions may be updated at any time, we don't want to maintain the list ourselves.